repair/repair.cc

```diff
@@ -1036,9 +1036,10 @@ static future<> repair_range(repair_info& ri, const dht::token_range& range) {
       }
       return ::service::get_local_migration_manager().sync_schema(ri.db.local(), neighbors).then([&neighbors, &ri, range, id] {
         return do_for_each(ri.cfs.begin(), ri.cfs.end(), [&ri, &neighbors, range] (auto&& cf) {
+            auto idx = std::distance(ri.cfs.begin(), std::find(ri.cfs.begin(), ri.cfs.end(), cf));
             ri._sub_ranges_nr++;
             if (ri.row_level_repair()) {
-                return repair_cf_range_row_level(ri, cf, range, neighbors);
+                return repair_cf_range_row_level(ri, cf, ri.table_ids[idx], range, neighbors);
```

Comment:
Again we should iterate only in table_ids, and if we want the name, grab it from the schema.

Reply:
The tables names and ids are converted once, the are fixed 1:1 mapping. I tried to iterate the index but do_for_each does not take do_for_each(boost::irange(0, size)). 

I will change the code to iterate table_ids and find the table name on the fly.